backends/coda/table/coda-table.js

```diff
@@ -4,6 +4,8 @@ import Coda from "../coda.js";
  * @category Coda
  */
 export default class CodaTable extends Coda {
+	static capabilities = { auth: true, put: false, upload: false };
```

Comment:
I wonder whether we should be explicit when defining capabilities or `undefined === false`? E.g., here, we have `upload: false`, but we don't specify it in, e.g., the `Basic` backend.

Reply:
The only reason I specified it explicitly here is because it should change once the Coda backend is implemented properly.

Reply:
Make sense. Thanks for the answer